pynest/nest/lib/hl_api_types.py

```diff
@@ -442,16 +446,30 @@ def index(self, node_id):
             If the node ID is not in the `NodeCollection`.
         """
         index = sli_func('Find', self._datum, node_id)
+
         if index == -1:
             raise ValueError('{} is not in NodeCollection'.format(node_id))
+
         return index
 
+    def __array__(self, dtype=None):
+        """ Convert the NodeCollection to a NumPy array. """
```

Comment:
One-line docstrings should have no blank lines either before or after the docstring (see https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).
```suggestion
        """Convert the NodeCollection to a NumPy array."""
```